src/backend/src/app.js

```diff
@@ -45,12 +49,22 @@ export async function init({ version, rootKey }) {
   log('Starting app...')
   log(`Device version is ${version}`)
 
-  // 1. Initialize Mapeo
+  const privateStorageDir = rnBridge.app.datadir()
+  const dbDir = join(privateStorageDir, DB_DIR_NAME)
+  const indexDir = join(privateStorageDir, INDEX_DIR_NAME)
+
+  if (!existsSync(dbDir)) {
```

Comment:
It's best practice to avoid checking existence before an action, see https://nodejs.org/api/fs.html#fs_fs_exists_path_callback, although in this case I can't imagine a case where it would cause an issue. However calling mkdirSync with `recursive: true` will work regardless of whether the folder already exists